Sources/NIOCrashTester/CrashTests.swift

```diff
@@ -0,0 +1,38 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import NIO
+import Foundation
+
+let group = MultiThreadedEventLoopGroup(numberOfThreads: 2)
+
+struct Foo {
+    let multiThreadedELGCrashesOnZeroThreads = CrashTest { requireCrashRegex in
```

Comment:
note to reviewers: this is how crash tests look. For each thing you'd like to test you add a

```swift
let whatYouWantToTest = CrashTest { requireCrashRegex in
    requireCrashRegex("^regex of what message you want to see")

    whatMakesItCrash()
}
```